### ISSUE TITLE
Add a backup option?

### ISSUE THREAD

**Issue by grajewski** [OpenLiveWriter/OpenLiveWriter#341]
Hello! First-time member of this community. I did a search to see if anyone has raised this as a new feature, but found nothing.

When using WLW 2012, I came across a 3rd party app called "Windows Live Writer Backup" (WLWB) created by Scott Lovegrove, but the last stable version dates from 2011.

I don't think that WLWB works with Open Live Writer, so I don't have any idea of how to backup OLW going forward.

Is this a feature or plug-in that may be developed in the not so distant future?

Thanks for considering this and thank-you to the developers for resurrecting WLW as an open-source option! :-)

**Comment by ScottIsAFool** [comment]
Correct, it doesn't work with OLW and it's on my list of things to do to update it so that it can be used.

**Comment by grajewski** [comment]
Scott, I am honored by a response by the developer himself. :-)

I look forward to future developments of WLWB and have just added your RSS feed to my Feedly app for Android in order to stay in touch.

**Comment by SteveP-UK** [comment]
@ScottIsAFool This was such a useful plugin under WLW - saved me many hours of faffing around when installing WLW on a new device. Happy to test something when you are ready.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
